src/olympia/constants/promoted.py

```diff
@@ -44,6 +47,7 @@ def __bool__(self):
 VERIFIED_ONE = PromotedClass(
     id=2,
     name=_('Verified - Tier 1'),
+    api_name='verified_tier_1',
```

Comment:
can we confirm this `api_name`? `sponsored` has been used for the codename elsewhere so maybe use it here?  (`sponsored` will be the signing state constant too for verified tier 1)

Reply:
https://github.com/mozilla/addons-server/pull/15221 for the signing changes.